include/inviwo/core/util/colorbrewer.h

```diff
@@ -543,6 +544,16 @@ IVW_CORE_API glm::uint8 getMaxNumberOfColorsForFamily(const Family &family);
  **/
 IVW_CORE_API std::vector<Family> getFamiliesForCategory(const Category &category);
 
+/**
+ * Returns a transfer function for the given parameters. 
+ * 
+ * @param category according to ColorBrewer2
+ * @param family color scheme name
+ * @param discrete will make each color constant instead of linearly varying inbetween colors.
+ * @param divergenceMidpoint in [0 1]. Only used when category is Diverging 
+ **/
+IVW_CORE_API TransferFunction getTransferfunction(const Category &category, const Family &family, glm::uint8 nColors, bool discrete, double divergenceMidPoint);
```

Comment:
This file is generated using a python script, see line 31

Reply:
Ah, updated script